src/addons/addonapi.cpp

```diff
@@ -90,6 +90,22 @@ void AddonApi::initialize() {
   if (s_constructorCallback) {
     s_constructorCallback(this);
   }
+
+  m_timer.setSingleShot(true);
+}
+
+void AddonApi::setTimedCallback(int interval, const QJSValue& callback) {
```

Comment:
```suggestion
void AddonApi::setTimeout(const QJSValue& callback, int interval) {
```

This design would mimic almost exactly the JavaScript setTimeout API, which I think is nice.

Reply:
IMO, I think this is a better name than the Javascript method. Switching the parameter order to match the setTimeout looks good.

Reply:
I also like the name `setTimedCallback` a little more. I'm not sure we need to line this function up with javascript usage.